flow/flat_buffers.h

```diff
@@ -250,6 +251,29 @@ struct vector_like_traits<std::set<Key, Compare, Allocator>> : std::true_type {
 		return v.begin();
 	}
 };
+template <class Key, class Compare, class Allocator>
+struct vector_like_traits<std::unordered_set<Key, Compare, Allocator>> : std::true_type {
+	using Vec = std::unordered_set<Key, Compare, Allocator>;
+	using value_type = Key;
+	using iterator = typename Vec::const_iterator;
+	using insert_iterator = std::insert_iterator<Vec>;
+
+	template <class Context>
+	static size_t num_entries(const Vec& v, Context&) {
+		return v.size();
+	}
+	template <class Context>
+	static void reserve(Vec&, size_t, Context&) {}
```

Comment:
It looks like std::unordered_set does have an appropriate reserve function that we can call: https://en.cppreference.com/w/cpp/container/unordered_set/reserve